site/content/en/docs/Examples/simple-genai-gameserver.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Build and Run a Simple Game Server that Connects to an Inference Server"
+linkTitle: "GenAI Game Server"
+date:
+publishDate:
+description: >
+  This example shows how to set up a simple game server that integrates with your inference server's GenAI endpoints. You can either interact with one GenAI endpoint via TCP, or set up two endpoints to "chat" to each other.
+---
+
+## Setting up the GenAI Inference Server
+
+You will need a separate GenAI inference server. This example uses the
+[Google for Games GenAI](https://github.com/googleforgames/GenAI-quickstart) as its inference server.
+This particular inference server has the request structure for the /chat endpoint:
+
+```
+type GenAIRequest struct {
+	Context         string  `json:"context,omitempty"`
+	Prompt          string  `json:"prompt"`
+}
+```
+
+### (Optional) Modify the GenAIRequest Structure
+
+If you need a different request structure for your GenAI endpoint, you will need to clone or fork the Agones repository and
+modify the above `GenAIRequest struct` in [main.go](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/main.go). Update `REPOSITORY` in the
+[agones/examples/simple-genai-server/Makefile](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/Makefile)
+to your own container registry and run `make build && make push` from within the
+`agones/examples/simple-genai-server` path. Then modify the `gameserver_*.yaml` to pull the image from
+your container registry. If you are making a series of changes you may also want to add
+`imagePullPolicy: Always` to the container image in `gameserver_*.yaml`.
+
+## Setting up Agones
+
+To set up the Game Servers on a different cluster than the GenAI inference server follow the
+instructions for [creating a cluster](https://agones.dev/site/docs/installation/creating-cluster/).
+
+If you set up a separate cluster for your game servers [install Agones](https://agones.dev/site/docs/installation/install-agones/).
+on that cluster, otherwise install Agones into your GenAI inference server cluster.
+
+## Setting up the Game Server
+
+To manually interact with GenAI endpoint via netcat, modify the
+[gameserver_manualchat.yaml](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/gameserver_manualchat.yaml) `GenAiEndpoint` value to your inference
+server's endpoint. Optionally, include the `GenAiContext` in your `gameserver.yaml` that will
```

Comment:
Replace `GenAiContext ` with `GEN_AI_CONTEXT`?

There is no `gameserver.yaml` in the `examples/simple-genai-server` directory. May I know which gameserver.yaml we are referring to?

Reply:
Here it's referring to the `gameserver_manualchat.yaml`, I'll add that in.